src/target/intrin_rule.cc

```diff
@@ -118,6 +119,22 @@ TVM_REGISTER_OP("tir.nearbyint")
 TVM_REGISTER_OP("tir.pow").set_attr<FLowerIntrinsic>("default.FLowerIntrinsic",
                                                      DispatchPureExtern<FloatSuffix>);
 
+PrimExpr DispatchFastErf(const PrimExpr& e) {
+  LOG(WARNING) << "fast_erf will be used instead of erf";
+  const CallNode* call = e.as<CallNode>();
+  ICHECK(call != nullptr);
+  ICHECK_EQ(call->args.size(), 1);
+  PrimExpr arg = call->args[0];
+  int bits = arg.dtype().bits();
+  PrimExpr res;
+  if (arg.dtype().is_float() && (bits == 16 || bits == 32)) {
+    res = topi::fast_erf_float_expr(arg, bits);
```

Comment:
i know it's from some legacy impl of metal, but can we structure the folder slightly better to avoid depending on topi? i.e. moving this method to tir instead.

Reply:
Thanks for the comment, reorganized to remove topi dep